contracts/test/avs/OmniAVS.t.sol

```diff
@@ -102,12 +109,16 @@ contract OmniAVS_Test is AVSBase, AVSUtils {
         validators = omniAVS.getValidators();
 
         // assert all operators have been registered
-        assertEq(validators.length, numOperators);
+        assertEq(validators.length, numOperators, "_testRegisterOperatorsWithAVS: all operators should be registered");
 
         // assert operator has initial stake
         for (uint32 i = 0; i < numOperators; i++) {
-            assertEq(validators[i].staked, initialOperatorStake);
-            assertEq(validators[i].delegated, 0);
+            assertEq(
+                validators[i].staked,
+                initialOperatorStake,
+                "_testRegisterOperatorsWithAVS: validator.staked should be initialOperatorStake"
+            );
+            assertEq(validators[i].delegated, 0, "_testRegisterOperatorsWithAVS: validator.delegated should be 0");
```

Comment:
added a bunch of these assert error messages because they help with debugging in this big ole tests. we should have them in more places